MaxText/configs/base.yml

```diff
@@ -44,20 +44,20 @@ int8_training: False
 # Global parameter scale needs to be a power of 2. If you want finer grained control of the model sizes 
 # then you should explicitly set base_embed_dim, base_num_heads, base_mlp_dim, base_num_decoder_layers and/or head_dim.
 global_parameter_scale: 1
-base_emb_dim: 2560
-base_num_heads: 8
-base_mlp_dim: 8192
-base_num_decoder_layers: 16
-head_dim: 256
+base_emb_dim: 4096
+base_num_heads: 32
+base_mlp_dim: 11008
+base_num_decoder_layers: 32
+head_dim: 128
 # activation functions are .
-mlp_activations: ["relu"]
+mlp_activations: ["silu"]
 dropout_rate: 0
 logits_via_embedding: True  # NOTE: this is True just for testing.
 # proj, minimal, full, or none
 remat_policy: 'full'
 scan_layers: True
 param_scan_axis: 1
-attention: 'flash' # Supported attention: flash, mha
+attention: 'mha' # Supported attention: flash, mha
```

Comment:
2cents of mine not related to this PR: it may be OK to not use flash attention by default. Also we should support more both flash and non flash attention. As we are also need flash_mha_gpu and flash_mha_tpu (this current `flash`).

Also to extend current code to support llama-70b and mixtral in future, we need (GQA and local attention).
And may need flash_gqa_gpu(tpu) and flash_sliding_widow extra.

For inference mode, we may need to show performance of MQA as well. So In fact we need a more attention layers.

Reply:
100% agreed on supporting all of these things in teh future. However, I think flash should be the default unless you have a specific concern. (Is that concern "GPU won't work out of the box" -- that is a good one!)

Reply:
reverted this back.